apps/backoffice-v2/src/common/components/atoms/IndeterminateCheckbox/IndeterminateCheckbox.tsx

```diff
@@ -0,0 +1,24 @@
+import { ComponentProps, FunctionComponent } from 'react';
+import { Checkbox_ } from '@/common/components/atoms/Checkbox_/Checkbox_';
+import { ctw } from '@/common/utils/ctw/ctw';
+
+export interface IIndeterminateCheckboxProps extends ComponentProps<typeof Checkbox_> {
+  indeterminate?: boolean;
+}
+
+export const IndeterminateCheckbox: FunctionComponent<IIndeterminateCheckboxProps> = ({
+  checked,
+  indeterminate,
+  className,
+  onCheckedChange,
+  ...rest
+}) => {
+  return (
+    <Checkbox_
+      className={ctw(className)}
+      checked={!checked && indeterminate ? 'indeterminate' : checked}
```

Comment:
Consider separate `checked` and `indeterminate`

Good [reference](https://mui.com/material-ui/react-checkbox/#indeterminate)